tracer/src/Datadog.Trace/Sampling/SamplingMechanism.cs

```diff
@@ -72,4 +74,27 @@ internal static class SamplingMechanism
     /// The only available sampling priority is <see cref="SamplingPriorityValues.UserKeep"/> (2).
     /// </summary>
     public const int SpanSamplingRule = 8;
+
+    public static string GetTraceContextDecisionMakerTagValue(int mechanism)
```

Comment:
Can we call this `GetSamplingMechanismTagValue()` or simply `GetTagValue()`?

I think `TraceContext` is redundant and `DecisionMaker` is the wrong name. "Decision maker" used to be the service name but we don't include that anymore in this tag, only the sampling mechanism.

Reply:
done in [cf86494](https://github.com/DataDog/dd-trace-dotnet/pull/4085/commits/cf86494ad39ef502551c6ae34c1b7f2e5a474735)